### ISSUE TITLE
[Bug]:  Unable to Use Claude-3-5-haiku-20241022 Model - Invalid Request Error

### ISSUE THREAD

**Issue by andressade** [danny-avila/LibreChat#4643]
### What happened?

When attempting to use the Claude-3-5-haiku-20241022 model for general conversation, the operation fails after 3 attempts with a 400 error

### Steps to Reproduce

Selected Claude-3-5-haiku-20241022 model
Attempted to have a normal conversation
Received error message

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
error.log:


{"level":"error","message":"[handleAbortError] AI response error; aborting request: Operation failed after 3 attempts: 400 {\"type\":\"error\",\"error\":{\"type\":\"invalid_request_error\",\"message\":\"prompt: Input should be a valid string\"}}","stack":"Error: Operation failed after 3 attempts: 400 {\"type\":\"error\",\"error\":{\"type\":\"invalid_request_error\",\"message\":\"prompt: Input should be a valid string\"}}\n    at AnthropicClient.processResponse (/app/api/app/clients/AnthropicClient.js:815:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AnthropicClient.sendCompletion (/app/api/app/clients/AnthropicClient.js:828:5)\n    at async AnthropicClient.sendMessage (/app/api/app/clients/BaseClient.js:572:24)\n    at async AskController (/app/api/server/controllers/AskController.js:125:20)\n    at async /app/api/server/routes/ask/anthropic.js:23:5"}


debug.log:


2024-11-05T20:11:26.303Z debug: [AnthropicClient]
{
  model: "claude-3-5-haiku-20241022",
  stream: true,
  // 3 stop_sequence(s)
  stop_sequences: ["||>","\n\nHuman:","<|diff_marker|>"],
  temperature: undefined,
    metadata.user_id: "abcxyz",
  top_p: undefined,
  top_k: undefined,
  // 1 prompt(s)
  prompt: [{"content":"test message","author":"\n\nHuman:"}],
  max_tokens_to_sample: 4096,
}
2024-11-05T20:11:26.305Z debug: Making request to https://api.anthropic.com/v1/complete
2024-11-05T20:11:26.511Z warn: User: abcxyz | Anthropic Request 1 failed: 400 {"type":"error","error":{"type":"invalid_request_error","message":"prompt: Input shou... [truncated]
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Already resolved on the latest build: https://github.com/danny-avila/LibreChat/pull/4629

**Comment by andressade** [comment]
problem still exist on 0c2a583d

**Comment by danny-avila** [comment]
![image](https://github.com/user-attachments/assets/8a3b1b9e-ae7a-4c2e-84db-e0343bb198c2)

**Comment by andressade** [comment]
![image](https://github.com/user-attachments/assets/21b7f32a-5be5-451e-8f7d-bf9c8cdcc206)

And I've restarted it 10 times... but still nothing :(

**Comment by danny-avila** [comment]
@andressade Are you using docker or npm? 

If docker: https://www.librechat.ai/docs/local/docker#update-librechat

If npm: https://www.librechat.ai/docs/local/npm#update-librechat

**Comment by andressade** [comment]
I finally tracked down the problem. I was using Docker and had hoped that after a "git pull" and docker-compose restart, the latest version would be in use. However, in docker-compose, the *.js files weren't mapped to the host; instead, the JavaScript files were being read from the container image. I didn't realize this right away.
I updated the Docker image, and the new model started working immediately :)
